### ISSUE TITLE
Hide matplotlib output

### ISSUE THREAD

**Issue by sfwolfphys** [davidesarra/jupyter_spaces#24]
I'm not sure if this is an issue or simply an operator error.  I'm trying to use jupyter spaces in conjunction with figures in matplotlib, and when I do that, the code spits out a bunch of matplotlib stuff.  It seems like the `%matplotlib inline` magic isn't taking (I run that in a previous cell in the global space.

Here is a screenshot of my output:
![image](https://user-images.githubusercontent.com/31363007/194880628-15efbf86-9d65-4719-a743-e69bd92dabad.png)

When I remove the `%%space prob1` line, the figure is shown directly:
![image](https://user-images.githubusercontent.com/31363007/194880995-087c017c-13cd-4df0-818a-d3dfb44db000.png)

Below is my code so you don't have to extract it from the image if you need it:

```
import numpy as np
import matplotlib.pyplot as plt

t = np.linspace(0,6*np.pi,500)
A = 1
gamma1 = -1/(2*np.pi)*np.log(0.75)
omegad1 = 1
phi1 = -np.arctan(gamma1/omegad1)
x1 = A*np.exp(-gamma1*t)*np.cos(omegad1*t+phi1)
gamma2a = 2*gamma1
phi2a = -np.arctan(gamma2a/omegad1)
x2a = A * np.exp(-gamma2a*t) * np.cos(omegad1*t + phi2a)

fig, ax = plt.subplots(1,1)
ax.plot(t,x1,color='black',label=r'$\gamma=${:.3f}'.format(gamma1)+r' $\phi_0=${:.3f}'.format(phi1))
ax.plot(t,x2a,'b-',label=r'$\gamma=${:.3f}'.format(gamma2a)+r' $\phi_0=${:.3f}'.format(phi2a))
ax.plot(t,A*np.exp(-gamma1*t),'g:')
ax.plot(t,-A*np.exp(-gamma1*t),'g:')
ax.plot(t,A*np.exp(-gamma2a*t),'b:')
ax.plot(t,-A*np.exp(-gamma2a*t),'b:')
ax.set_xticks(ticks=np.linspace(0,6*np.pi,7))
ax.set_xticklabels([])
ax.set_yticks(np.linspace(-1,1,5))
ax.set_yticks(np.linspace(-1,1,9),minor=True)
ax.set_xlabel('time')
ax.set_ylabel('displacement (m)')
plt.grid()
plt.legend()
plt.show()
```

**Comment by davidesarra** [comment]
@sfwolfphys, thanks for opening an issue 👍 

For reference here is a compact example to reproduce the behavior:

```
In [1]: %load_ext jupyter_spaces

In [2]: %%space name_of_the_space
   ...: 1
   ...: 2
   ...: 
Out[2]: 1
Out[2]: 2
```

Essentially every expression is rendered as an output.

It seems that this is a regression bug introduced in version `0.2.0` when trying to automatically render the last cell result.

As a workaround at least for now I suggest installing version `0.1.2` which is not affected.

```bash
pip install jupyter_spaces==0.1.2
```

Please see the [documentation for that version](https://github.com/davidesarra/jupyter_spaces/tree/0.1.2).

Apologies for the problem and thanks again for opening the issue!

**Comment by sfwolfphys** [comment]
Thanks, that seems to work for me.

**Comment by davidesarra** [comment]
This issue has been fixed by #29. Whilst waiting for a new release the fixed version can be installed from source:

```bash
pip install git+https://github.com/davidesarra/jupyter_spaces.git
```

**Comment by davidesarra** [comment]
@sfwolfphys The problem should now be solved: only the last output of the cell is shown.

You can install the latest version from PyPI:

```bash
pip install "jupyter-spaces>=0.3.0"
```

Thanks again 👍
